src/main/resources/swagger/api-docs.yaml

```diff
@@ -864,6 +864,103 @@ paths:
             - email
             - profile
 
+  /api/workflows/{version}/{id}/releaseHold:
+    post:
+      x-passthrough: true
+      x-passthrough-target: cromiam
+      summary: Request Cromwell to release the hold on a workflow. It will switch the status of a workflow
+        from ‘On Hold’ to ‘Submitted’ so it can be picked for running. For instance this might be necessary
+        in cases where you have submitted a workflow with workflowOnHold = true.
+      parameters:
+        - name: version
+          description: API Version
+          required: true
+          type: string
+          in: path
+          default: v1
+        - name: id
+          description: Workflow ID
+          required: true
+          type: string
+          in: path
+      tags:
+        - CromIAM Workflows (for Admin)
+      responses:
+        '200':
+          description: Successful Request
+          schema:
+            $ref: '#/definitions/WorkflowStatusResponse'
+        '400':
+          description: Malformed Workflow ID
+        '403':
+          description: Malformed Request
```

Comment:
403 is usually Forbidden - is it really malformed request for this API? If that's what cromiam returns, then ok

Reply:
this is from the CromIAM swagger